recover/recovery_test.go

```diff
@@ -158,15 +158,11 @@ func TestRecoverer_RecoverFromXMRMakerSecretAndContract_Claim(t *testing.T) {
 }
 
 func TestRecoverer_RecoverFromXMRMakerSecretAndContract_Claim_afterTimeout(t *testing.T) {
-	// if testing.Short() {
-	// 	t.Skip() // TODO: fails on CI with "no contract code at address"
-	// }
-
 	keys, err := pcommon.GenerateKeysAndProof()
 	require.NoError(t, err)
 
 	claimKey := keys.Secp256k1PublicKey.Keccak256()
-	addr, contract, swapID, swap := newSwap(t, claimKey, [32]byte{}, false)
```

Comment:
@noot The boolean field at the end is `setReady`. If it's false, the test was failing with `VM Exception while processing transaction: revert too early to claim!`.  Here's the relevant solidity code:
```
    function claim(Swap memory _swap, bytes32 _s) public {
[...]
        require((block.timestamp >= _swap.timeout_0 || swapStage == Stage.READY), "too early to claim!");
```
... so the test was failing because the `swapStage` was never set to `READY`. Should this test have worked before my change?

Reply:
I think it should have, because the recoverer should have waited until past t0 to claim. I don't really see anywhere in your changes that would cause this to fail though